dev/com.ibm.ws.anno/src/com/ibm/ws/annocache/targets/cache/internal/TargetCacheImpl_Reader.java

```diff
@@ -335,6 +420,19 @@ public TargetsReader() {
          */
         public abstract String getTableVersion();
 
+        /**
+         * Answer the versions which are accepted within the version
+         * line, "Version: &lt;tableVersion&gt;".</p>
+         *
+         * <p>Used by {@link #handleHeader()} to validate the table which
+         * is being parsed.</p>
+         *
+         * @return The versions which are accepted on the version line.
+         */
+        public String[] getTableVersions() {
+            return new String[] { getTableVersion() };
```

Comment:
Should this method be abstract like getTableVersion?

Reply:
Most tables have a single version.  This provides a default implementation for those table types.